### ISSUE TITLE
licensing questions

### ISSUE THREAD

**Issue by urlordjames** [Mitchellbrine/DiseaseCraft-Remastered#6]
I have a few questions in regards to licensing.  According to the CurseForge page in the pre-1.18.2 section (https://www.curseforge.com/minecraft/mc-mods/diseasecraft), the old version of DiseaseCraft was released under GPL v3.  The GitHub repository suggests that the license is actually LGPL v3 (https://github.com/Mitchellbrine/DiseaseCraft-2.0/blob/master/LICENSE).  I'm unsure which license the pre-1.18.2 version was released under, can you please clarify?  Additionally, if the 1.18.2 version is a modified version of the pre-1.18.2 code, then the GPL/LGPL also applies to that modified copy (see https://www.gnu.org/licenses/gpl-faq.en.html#GPLRequireSourcePostedPublic).  If the two versions are distinct, you may want to specify in the README that the 1.18.2 version is a rewrite of the pre-1.18.2 version.

In summary:
- What license was the pre-1.18.2 version released under? (GPL v3 or LGPL v3)
- Is the 1.18.2 version a modified version of the pre-1.18.2 version?  If not, may want to specify this in a README.

Thank you for your continued dedication in supporting this mod after all these years!

**Comment by Mitchellbrine** [comment]
“DiseaseCraft: Remastered” has been rewritten completely from scratch. I did not utilize a single line of code from previous versions (save for any little holdovers in how Forge runs). It is the fourth iteration of this mod, and returns to the same license as originally released (All Rights Reserved). 

VERSION 1 was All Rights Reserved (as VERSION 4 is now), which means I did not release VERSION 1’s code and will not be releasing the source code for VERSION 4 yet (I plan on doing so when I feel it’s stable and presentable).
VERSIONS 2 and 3 (which are the version in the other repository that has not been committed to in years) are under LGPL v3. VERSION 2 was also a complete rewrite from scratch, while VERSION 3 was written from that.

All that to say, CurseForge was a mishap on my part (versions 2 and 3 were under LGPL not GPL), and the current version is distinct from the other versions of the mod.

Thank you for asking for the clarification, I’ll fix the CurseForge page.

**Comment by urlordjames** [comment]
Thank you for clarifying!
